.github/workflows/reusable-container-release.yml

```diff
@@ -10,6 +10,10 @@ on:
         type: string
         required: false
         default: Dockerfile
+    secrets:
+      # analytical-platform-container-test-slack-webhook-url:
+      ANALYTICAL_PLATFORM_CONTAINER_TEST_SLACK_WEBHOOK_URL:
```

Comment:
This doesn't have to match the secret, for example look at this https://github.com/ministryofjustice/analytical-platform-github-actions/blob/0b8979a3ad555f77617f67b2e6afa8f66672ada1/.github/workflows/reusable-scheduled-container-scan.yml#L13-L15

it doesn't match the secret casing here https://github.com/ministryofjustice/analytical-platform-github-actions/blob/0b8979a3ad555f77617f67b2e6afa8f66672ada1/docs/usage/workflows/scheduled-container-scan.md?plain=1#L30-L31